tests/profiles/helm/values-istio-default.yaml

```diff
@@ -1,2 +1,9 @@
 # default profile will use all the settings in global.yaml
+global:
+  istioNamespace: istio-system
```

Comment:
This should not be required, it's translated from IstioControlPlane API.

Reply:
This is values file for `hem template ...`